packages/ember-testing/lib/adapters/qunit.js

```diff
@@ -18,6 +18,6 @@ export default Adapter.extend({
     QUnit.start();
   },
   exception(error) {
-    ok(false, inspect(error));
+    ok(false, inspect(error)); // How do we access assert.ok here?
```

Comment:
@rwjblue Should I use `QUnit.assert.ok` here instead?

Reply:
`QUnit.config.current.assert.ok`